### ISSUE TITLE
favicon is invisible in the dark-themed browser

### ISSUE THREAD

**Issue by 532910** [jitsi/jitsi-meet#6182]
![image](https://user-images.githubusercontent.com/14617699/79669402-e3486600-81c3-11ea-9047-dd2422f0c016.png)

**Comment by 532910** [comment]
Is any update there?

**Comment by 532910** [comment]
must be fixed

**Comment by Elaborendum** [comment]
Are there any news on this one? It's really hard to quickly find the tab when there are lots open.

**Comment by saghul** [comment]
The blue doesn't look all that great on dark TBH. Let me get you an SVG with the logo so you can work on the light / dark icon. Thanks for volunteering!

**Comment by saghul** [comment]
[Jitsi logo.svg.zip](https://github.com/jitsi/jitsi-meet/files/7258426/Jitsi.logo.svg.zip)

Here you go!

**Comment by WillowP** [comment]
Sweet! Can you assign the issue to me? I'll try to have a look this weekend :)

**Comment by WillowP** [comment]
Quick update: I'm thinking I'll follow the second answer on the SO link above, and also add the "mask-icon" for Safari as described in this [article](https://medium.com/swlh/are-you-using-svg-favicons-yet-a-guide-for-modern-browsers-836a6aace3df).

**Comment by saghul** [comment]
> Sweet! Can you assign the issue to me? I'll try to have a look this weekend :)

We don't do issue assignments as such, since you volunteered, it's yours!



> Quick update: I'm thinking I'll follow the second answer on the SO link above, and also add the "mask-icon" for Safari as described in this [article](https://medium.com/swlh/are-you-using-svg-favicons-yet-a-guide-for-modern-browsers-836a6aace3df).

Sounds good!

**Comment by 532910** [comment]
I'm for two icons for dark and white themed browsers. Github does so:
![image](https://user-images.githubusercontent.com/14617699/135715232-fc6cee75-b091-4158-b9a5-f51bbba257ef.png)
![image](https://user-images.githubusercontent.com/14617699/135715237-dc8a1805-8725-47d9-a5f2-8e0039ea3ab2.png)

**Comment by WillowP** [comment]
Quick update: though the implementation of this should be simple, the getting set up to test changes to `index.html` is harder than I expected. See my communication here https://community.jitsi.org/t/changes-for-meta-tag-in-title-html-not-reflected/71604/4

It might be nice to add some documentation on that. I'll try to keep track of what I discover as I get set up for testing this, and make a separate PR with docs for that.

**Comment by shane215** [comment]
Hey @WillowP, 
I'm new to this but happy to get stuck in and help on this issue.

Also I'm curious, why is this tagged as 'accessibility'?

**Comment by WillowP** [comment]
Hey @shane215 , thanks!

You can see what I've done so far at https://github.com/WillowP/jitsi-meet/tree/feature/svg-favicon

I've tested the svg file I made and it seems to be working.

I still want to add the mask icon as described here https://medium.com/swlh/are-you-using-svg-favicons-yet-a-guide-for-modern-browsers-836a6aace3df

The reason I haven't done this yet is I haven't had time to figure out the solution to testing described at https://community.jitsi.org/t/changes-for-meta-tag-in-title-html-not-reflected/71604/4

If you figure that out, I'd love to see a step-by-step guide on how you made it happen. Might be a nice thing to add to the developer docs.

I think it's tagged as accessibility because it's a contrast issue. People with visual impairments would not be able to see the favicon at all on dark, even though others might be able to see it.

**Comment by shane215** [comment]
Thanks @WillowP, I'll look into that community post when I can.
Same here, time is very limited for this stuff.

**Comment by shane215** [comment]
I wrote up an explanation of [how Jitsi Meet bypasses the webpack-dev-server proxy](https://community.jitsi.org/t/how-does-jitsi-meet-bypass-webpack-dev-server-proxy/109513).

So to bypass the proxy for `/index.html`, I think the following code should be added to `devServerProxyBypass`:

`if (path == '/index.html'){
        return path;
 }`

**Comment by shane215** [comment]
> 

I added a console log to print out `path is: ${path}` to see what paths are being passed into `devServerProxyBypass`, and I got the below output in which I don't see `index.html`. 

So I'm not sure how we are going to check when a request is trying to load `index.html`.

path is: /
path is: /css/all.css
path is: /libs/external_connect.js
path is: /libs/lib-jitsi-meet.min.js
path is: /libs/do_external_connect.min.js
path is: /libs/app.bundle.min.js
path is: /pwa-worker.js
path is: /images/apple-touch-icon.png
path is: /libs/olm.wasm
path is: /images/favicon.ico
path is: /images/app-store-badge.png
path is: /images/watermark.svg
path is: /images/google-play-badge.png
path is: /images/f-droid-badge.png
path is: /images/welcome-background.png
path is: /sounds/e2eeOff.mp3
path is: /sounds/e2eeOn.mp3
path is: /sounds/asked-unmute.mp3
path is: /sounds/joined.wav
path is: /sounds/left.wav
path is: /sounds/incomingMessage.wav
path is: /sounds/knock.mp3
path is: /sounds/reactions-thumbs-up.mp3
path is: /sounds/reactions-thumbs-up.mp3
path is: /sounds/reactions-thumbs-up.mp3
path is: /sounds/reactions-applause.mp3
path is: /sounds/reactions-applause.mp3
path is: /sounds/reactions-applause.mp3
path is: /sounds/reactions-laughter.mp3
path is: /sounds/reactions-laughter.mp3
path is: /sounds/reactions-laughter.mp3
path is: /sounds/reactions-surprise.mp3
path is: /sounds/reactions-surprise.mp3
path is: /sounds/reactions-surprise.mp3
path is: /sounds/reactions-boo.mp3
path is: /sounds/reactions-boo.mp3
path is: /sounds/reactions-boo.mp3
path is: /sounds/reactions-crickets.mp3
path is: /sounds/reactions-crickets.mp3
path is: /sounds/reactions-crickets.mp3
path is: /sounds/reactions-raised-hand.mp3
path is: /sounds/liveStreamingOff.mp3
path is: /sounds/liveStreamingOn.mp3
path is: /sounds/recordingOff.mp3
path is: /sounds/recordingOn.mp3
path is: /sounds/rejected.wav
path is: /sounds/rejected.wav
path is: /sounds/outgoingRinging.wav
path is: /sounds/outgoingStart.wav
path is: /sounds/talkWhileMuted.mp3
path is: /sounds/noisyAudioInput.mp3
path is: /sounds/noAudioSignal.mp3
path is: /pwa-worker.js

**Comment by shane215** [comment]
I'm still stuck with how to test changes to /index.html on localhost.

I created a new community topic to ask for help:
https://community.jitsi.org/t/how-to-test-changes-to-index-html-on-localhost/110831

**Comment by shane215** [comment]
I figured out a way to load my local index.html as explained [here](https://community.jitsi.org/t/how-to-test-changes-to-index-html-on-localhost/110831?u=shane.215).

Opened PR #11130.
